Cargo.toml

```diff
@@ -114,6 +114,7 @@ serde_json = "1.0.105"
 serde_with = "3.0.0"
 sha2 = "0.10.7"
 sha3 = "0.10"
+smallvec = "1.13.1"
```

Comment:
Didn't we agree that this dependency should be removed?

Reply:
Oh, that was just a change coming from PR #1898. I've changed this PR to reflect that this is to be merged _after_ that one has been merged.